ampersand-chart.js

```diff
@@ -609,42 +601,50 @@
         .style('display', undefined);
     },
     renderData: function() {
-      var data = this.model._data.models;
+      var self = this;
+      setTimeout(function() {
+        var data = self.model._data.models;
```

Comment:
we are using `self.model` quite a bit here. We should cache it for perf to prevent redundant lookups